### ISSUE TITLE
build problem

### ISSUE THREAD

**Issue by antst** [git-developer/RCSwitchFirmata#1]
Which versions of ConfigurableFirmata and Firmata do you use?

I can't build it with any combinations I tried so far there are some errors :(

**Comment by git-developer** [comment]
RCSwitch was developed against ConfigurableFirmata 2.6.0 (2014-03-08). The code was commited on branch `configurable_dev` which can still be found on the [forked repo](https://github.com/git-developer/arduino/tree/configurable_dev).

You may also be interested in the corresponding commit in the [original repo](https://github.com/firmata/arduino/commit/53e50a69283c872732759d04dcc97c86afde1564) and the [pull request](https://github.com/firmata/arduino/pull/131).

**Comment by antst** [comment]
Thanks! I sorted out issues. I also had problem with my remote, but it turned out to be older version of RCSwitch, included into RCSwitchFirmata.
After all, I was able to build it with master branches of RCSwitch, Firmata, ConfigurablFirmata. And it works perfectly, thanks! 

Although, now I have another issue. While RCSwitch has no issues with my remotes (IMPULS) and receive/send signals, they use not fully compliant IT protocol, and FHEM module is confused :)
